database/migrations/wallet-migration.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+      await queryInterface.createTable('wallets', 
+      {
+        id: {
+          type: Sequelize.UUID,
+          primaryKey: true,
+        },
+        user_id: {
+            type: Sequelize.STRING,
+            allowNull: false,
```

Comment:
This should reference the id in the users model since it is a foreign key See image below
![image](https://user-images.githubusercontent.com/60648438/207048404-1d11a90e-96d0-4398-84c0-db9fad395767.png)

Reply:
Also the type should be uuid since it will contain the same info as the user table id which will be a uuid too